### DISCUSSION TITLE
Data grid

### DISCUSSION THREAD

**Discussion by ThePatrickMartin** [cnjinhao/nana#591] [Q&A] [answered]
Any advice for achieving grid functionality?

**Comment by qPCR4vir** [comment]
Hi,
you mean [grid](https://github.com/cnjinhao/nana/wiki/Div-Text#grid) for [place]( http://nanapro.org/en-us/documentation/page.php?u=/utilities/place) or more like a table: [table/listbox](http://nanapro.org/en-us/documentation/page.php?u=/widgets/listbox)?

**Comment by ThePatrickMartin** [comment, marked as answer]
It seems the listbox fits the bill - for displaying tabular data.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
